script/build.js

```diff
@@ -125,11 +125,9 @@ smith.metadata({ buildtype: options.buildtype });
 const ignore = require('metalsmith-ignore');
```

Comment:
Wanted to get this merged without incurring another CI run. The changes you suggested are now part of #5923.